lib/bbox.dart

```diff
@@ -0,0 +1,3 @@
+library bbox;
```

Comment:
our namespaces start with `turf_`, could you change it to `turf_bbox`?

Reply:
Done